### DISCUSSION TITLE
Q-Select, buttons in after-options not reachable with keyboard navigation.

### DISCUSSION THREAD

**Discussion by danielwestin** [quasarframework/quasar#15787] [General - Components / Directives / etc]
Hello. 
I have a multi-select, q-select with q-checkboxes rendered. Instead of listen to a change on the q-checkbox i render two buttons in the q-select after-options slot. One for applying the selected values and one for clearing. Unfortunately i am not able to reach these buttons  with keyboard navigation. Does anyone have a solution for this issue? 

As mentioned in the docs i can easily navigate the options with keyboard arrows and tab key bring me to next focusable element on the DOM. I would like to have some sort of way to react elements in after-options if they are focusable.

**Comment by pdanpdan** [comment]
can you please make a codepen?

**Reply by danielwestin** [reply]
I created this one. Not the nicest but makes my point i think.

https://codepen.io/danielpetra/pen/BaqJNOo

**Comment by pdanpdan** [comment]
Yes, you are right, I now see what you mean - in QSelect the focus stays inside the QField so you cannot focus the buttons

I'll try to think of something

**Reply by danielwestin** [reply]
Thank you!

**Reply by danielwestin** [reply]
Cant seem to work it out. Have you come up/figured out with any possible solution?
